MiqoCraftCore/MiqoCraftCore.cs

```diff
@@ -104,7 +104,7 @@ public static string GetCraftingPreset(string gatheringType, string iSlot, FFXIV
                     nodeName = iGatheredItem.Name;
 
                     {
-                        nodeName = nodeName + "\",\"Crystal\",\"Shard";
+                        nodeName = "{" + nodeName + "}\",\"" + nodeName + "\",\"Cluster\",\"Crystal\",\"Shard\",\"a\",\"b\",\"c\",\"d\",\"e\",\"f\",\"g\",\"h\",\"i\",\"j\",\"k\",\"l\",\"m\",\"n\",\"o\",\"p\",\"q\",\"r\",\"s\",\"t\",\"u\",\"v\",\"w\",\"x\",\"y\",\"z";
```

Comment:
Idk why, but some times if i don't include the nodeName without {} miqo just ignore? until it have enough gp to do the rotation, so i add twice.
About the alphabet, it is just a fail safe for rouge nodes, like gold ore is a two in one grid that has gold ore and another spoilled.